src/index.jsx

```diff
@@ -24,55 +22,42 @@ ReactDOM.render(
     <CookiesProvider>
       <Router>
         <Routes>
-          <Route element={<Layout isAdmin={false} />}>
+          <Route element={<Layout isAdmin />}>
             <Route path="/pickup-box-form" element={<PickupBoxFormPage />} />
             <Route path="/relocate-box-form" element={<RelocateBoxFormPage />} />
           </Route>
```

Comment:
The pickup box form and relocate box forms are not admin pages, so please set the layout to not be the admin layout since doing is breaks the app because it is checking for a logged in user but no one is logged in when navigating to these forms.

Reply:
changed!